test/deployconfig.jl

```diff
@@ -140,7 +140,7 @@ end
     with_logger(NullLogger()) do
         # Regular tag build with GITHUB_TOKEN
         withenv(
-            "GITHUB_EVENT_NAME" => "push",
+            "GITHUB_EVENT_NAME" => "push", "GITHUB_SERVER_URL" => "github.com",
```

Comment:
Let's keep this to one per line?
```suggestion
            "GITHUB_EVENT_NAME" => "push",
            "GITHUB_SERVER_URL" => "github.com",
```
and then the same everywhere below.